mp/src/game/server/neo/neo_player.cpp

```diff
@@ -2468,11 +2468,42 @@ int	CNEO_Player::OnTakeDamage_Alive(const CTakeDamageInfo& info)
 				++iDamage;
 				flDmgAccumlator -= 1.0f;
 			}
+
+			// Mirror team-damage
+			const bool bIsTeamDmg = (attackerIdx != entindex() && attacker->GetTeamNumber() == GetTeamNumber());
+			const float flMirrorMult = NEORules()->MirrorDamageMultiplier();
+			if (flMirrorMult > 0.0f && bIsTeamDmg)
```

Comment:
`flMirrorMult` could be moved inside the `bIsTeamDmg` condition scope such that `bIsTeamDmg` is evaluated first, so we can skip calling `NEORules()->MirrorDamageMultiplier()` entirely for non-team damage cases where it is not needed.